src/run.ts

```diff
@@ -55,11 +57,16 @@ async function runLint(lintPath: string): Promise<void> {
     throw new Error(`please, don't change out-format for golangci-lint: it can be broken in a future`)
   }
 
+  const workingDirectory = path.resolve(core.getInput(`working-directory`))
+  if (!fs.existsSync(workingDirectory) || !fs.lstatSync(workingDirectory).isDirectory()) {
+    throw new Error(`working-directory (${workingDirectory}) was not a path`)
+  }
+
   const cmd = `${lintPath} run --out-format=github-actions ${args}`.trimRight()
   core.info(`Running [${cmd}] ...`)
```

Comment:
let's add `workingDirectory` to this or similar log line, please

Reply:
Done!